docs/reference/3-admin-api/admin-api-2.x/technical-articles/quick-start-calling-admin-api-2x-using-python.md

```diff
@@ -2451,6 +2451,314 @@ def import_applications(
 
 As a result, we will obtain multiple 201 Status Code
 
+## Api Clients
+
+### Create ApiClient for a given Application
+
+To create an ApiClient, we use the POST verb, and we will pass it a dictionary
+with the values to store, an example of payload for this case could be the
+following.
```

Comment:
From a new reader's perspective... what is an ApiClient? Why would I want to this? There may be existing ODS/API documentation that you could link to here for more information.

A short answer: 

```
The `apiClients` endpoint manages client credentials for accessing the Ed-Fi ODS/API.
```